app/Http/Controllers/PollController.php

```diff
@@ -156,8 +156,8 @@ public function showLite($poll)
     public function update(Request $request, Poll $poll)
     {
         $isPollAdmin = $poll->users->find(Auth::id())->pivot->admin === 1;
-        $ratings = DB::select('select * from poll_ratings where poll_id = ? AND user_id = ?', [$poll->id, Auth::id()]);
-        $hasRatings = count($ratings) > 0;
+        $count_ratings = DB::select('select count(*) as count from poll_ratings where poll_id = ? AND user_id = ?', [$poll->id, Auth::id()])->count;
```

Comment:
Please do not use snake case for php variable names

Reply:
Done